test/testinput/bufr_mhs.yaml

```diff
@@ -40,7 +57,41 @@ observations:
         - name: "nchans"
           size: variables/radiance.ncols
 
+      globals: 
+        - name: "sensorCentralFrequency@MetaData"
+          type: string
+          value: "89.0, 157.0, 183.311 +- 1.00, 183.311 +- 3.00, 190.311"
```

Comment:
@cyberbass 
I see that you want to put **sensor/channel-related information** under the global attribute.  Is this the direction we are heading? 

If so, we will need to add polarization (for MW) and original channel numbers (for all sensor types).

Reply:
Emily, in IODA variable naming convention, MetaData sheet, I created a variable called `sensorPolarizationDirection` that could do the job here.  But, now that I see what Nan typed in for this example, it reveals a small issue.  I had envisioned a list of frequencies in which each channel is a different freq, as is more typical on many platforms.  But this example has different tolerance ranges on the same freq (183GHz +- ??).  So perhaps another new addition is needed for channel tolerance or similar, because I wasn't expecting we would want to use strings to represent what are mostly floats.

Reply:
I think what I have there is just an example, we can fix it after discussion and I will update this PR.

Reply:
@cyberbass @gthompsnJCSDA @rmclaren 
In our re-engineering meeting today, I mentioned to @rmclaren that we will need the unit information for variables defined under global attributes. 
For example, 
the  globals: 
        - name: "sensorCentralFrequency@MetaData"
          type: string
          value: "89.0, 157.0, 183.311 +- 1.00, 183.311 +- 3.00, 190.311"

We need a unit (GHz) for this variable.  For IR, the unit could be wavelength (cm or micron).  

Another question I have is the data type for sensorCentralFrequency@MetaData.
The type is string.  This means that the users have to convert it to float type to use in numerical operations in UFO.  Can we define this variable directly to float type?

Reply:
@emilyhcliu I think we can define that directly into float type or float vectors but we can't add unit information to it. 
From the IODA global attributes examples I don't see there is a way to add units to a float type of global attributes: 

https://github.com/JCSDA-internal/ioda/blob/develop/src/engines/Examples/Basic/02-Attributes.cpp

Unless @rmclaren can think of a way to get around it.

Reply:
Nan:  can we just change this from a string of frequencies into a vector of floating point values, even if the 183GHz is repeated?  I want to see this PR approved asap.  I suggest we sideline the discussion of ways to deal with polarization and tolerance or other "edge cases" until later.  This is needing to get done and we can come back to those edge cases with different solutions later.  Other variables and/or attributes will be needed.

Reply:
@gthompsnJCSDA Yeah I would like to do that since we should get this PR in ASAP and to fully finish the discussion here may require some more time.

Reply:
@gthompsnJCSDA @cyberbass 
We can complete this PR as is.
We can open a new issue to continue the discussion.

Reply:
I just pushed the change to the yaml and the testoutput file in. I think we should be good to go.